packages/app/src/rhdh.json

```diff
@@ -0,0 +1,4 @@
+{
+  "rhdhVersion": "1.0.0",
```

Comment:
```suggestion
  "rhdhVersion": "1.1.0",
```

Reply:
could this be inherited from the root package.json? https://github.com/janus-idp/backstage-showcase/blob/main/package.json#L3

Reply:
No because it lives outside of the `packages/app/src` scope.

Reply:
ok, could this file be generated from package.json then? like `yarn build` just copies one line from the root package.json into this file?

also if the file is called rhdh.json and the field is called rhdhversion, then it's not really friendly for reskinning by a downstream consumer

maybe we want product.json and just `version` ?